src/client/dfs/duns.c

```diff
@@ -582,6 +592,49 @@ duns_create_path(daos_handle_t poh, const char *path, struct duns_attr_t *attrp)
 			rc = daos_cont_create(poh, attrp->da_cuuid, prop, NULL);
 			daos_prop_free(prop);
 		}
+
+		if (rc == -DER_SUCCESS && backend_dfuse) {
```

Comment:
im a little confused here. why would this be based on failure of the container create call?
it seems in most cases, this will not get executed even if the backend is dfuse. maybe im missing something.

Reply:
this is run based on success of the call.

Reply:
oops sorry, i didn't read this right